### PR TITLE
Scala 2.12.12

### PR DESCRIPTION
**Pull request by takayahilton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@ language: scala
 sudo: false
 jdk: openjdk8
 scala:
-- 2.12.11
+- 2.12.12
 - 2.13.3
 script:
-- sbt "++ ${TRAVIS_SCALA_VERSION}" "all scalafmtCheck test:scalafmtCheck scalafmtSbtCheck" test "scalaParser/testOnly scalaparser.SnippetSpec"
+- sbt "++ ${TRAVIS_SCALA_VERSION}" "all scalafmtCheck test:scalafmtCheck scalafmtSbtCheck" test
```
Comment (xuwei-k) [inline .travis.yml:8]: why changed? 🤔

**Review by xuwei-k — CHANGES_REQUESTED** [review]
please revert unnecessary format and build file changes

**Comment by takayahilton** [comment]
>format and build file changes

I ran `sbt" all scalafmtCheck test: scalafmtCheck scalafmtSbtCheck "test" scalaParser / testOnly scalaparser.SnippetSpec "` locally where this change was automatically included.
Anyway, It has nothing to do with this update, I will send it in another PR

**Review by xuwei-k — APPROVED** [review]

**Comment by sirthias** [comment]
Thank you, @xuwei-k! 👍

**Pull request merged**